dipy/segment/clustering.py

```diff
@@ -416,7 +417,7 @@ class QuickBundles(Clustering):
     metric : str or `Metric` object (optional)
         The distance metric to use when comparing two streamlines. By default,
         the Minimum average Direct-Flip (MDF) distance [Garyfallidis12]_ is
-        used and requires streamlines to have the same number of points.
+        used and streamlines are automatically resampled so they have 18 points.
```

Comment:
@MarcCote you can use the same default parameter in what was in the initial Quickbundles algorithm. If not as @arokem suggests you have to explain what will be the effect of the changes and give examples. 
I really like this new design :+1: 
```python
feature = ResampleFeature(nb_points=42)
metric = AveragePointwiseEuclideanMetric(feature)
qb = QuickBundles(threshold=10., metric=metric)
clusters = qb.cluster(streamlines)
```